src/pkg/inspection/controller.go

```diff
@@ -331,6 +333,21 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
 		}
 	}
 
+	// Read config from InspectionPolicy, send assessment reports to Governor api if governor enabled.
+	if policy.Spec.Inspection.Assessment.Governor.Enabled {
+		governorConfig := policy.Spec.Inspection.Assessment.Governor
+
+		if governorConfig.ClusterID == "" || governorConfig.URL == "" || governorConfig.APIToken == "" {
```

Comment:
We can be more specific with exact error.
`
if governorConfig.ClusterID == "" {
log.Error("GovernorExporter: Missing  ClusterID")
} 

if governorConfig.URL == "" {
} 

if governorConfig.APIToken == "" {
}
`